src/main/java/com/telefonica/jwk/Entrypoint.java

```diff
@@ -27,28 +26,52 @@
 public class Entrypoint {
   private static final String DEFAULT_KEY_SIZE = "2048";
   private static final Options OPTIONS = new Options();
-  private static final JsonElement JSON = new JsonParser();
+  private static final JsonParser JSON = new JsonParser();
   private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+  private static String size = null;
+  private static KeyUse keyUse = null;
+  private static Algorithm algorithm = JWEAlgorithm.RSA_OAEP_256;
+  private static boolean keySet = true;
```

Comment:
same with this. It was better as local var

Reply:
entonces tendría que meter un objeto para obtener el resultado del parseo ...